src/netex_france/exporter.rs

```diff
@@ -0,0 +1,113 @@
+// Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+// details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>
+
+//! Exporter for Netex France profile
+use crate::{model::Model, Result};
+use chrono::prelude::*;
+use minidom::Element;
+use std::{
+    convert::AsRef,
+    fmt::{self, Display, Formatter},
+    fs::File,
+    path::Path,
+};
+
+const NETEX_FRANCE_STOPS_FILENAME: &'static str = "arrets.xml";
+
+enum VersionType {
+    Stops,
+}
+
+impl Display for VersionType {
+    fn fmt(&self, fmt: &mut Formatter) -> std::result::Result<(), fmt::Error> {
+        use VersionType::*;
+        match self {
+            Stops => write!(fmt, "ARRET"),
+        }
+    }
+}
+
+/// Struct that can write an export of Netex France profile from a Model
+pub struct Exporter {
+    _model: Model,
+    participant_ref: String,
+}
+
+// Publicly exposed methods
+impl Exporter {
+    /// Build a Netex France profile exporter from the model.
+    /// `path` is the expected output Path where the Netex France is going to be
+    /// written. It should be a folder that already exists.
+    pub fn new(_model: Model, participant_ref: String) -> Self {
+        Exporter {
+            _model,
+            participant_ref,
+        }
+    }
+
+    /// Actually write `model` into `path` as a Netex France profile.
+    pub fn write<P>(&self, path: P) -> Result<()>
+    where
+        P: AsRef<Path>,
+    {
+        self.write_stops(path)?;
+        Ok(())
+    }
+}
+
+// Internal methods
+impl Exporter {
+    // Include 'stop_frame' into a complete NeTEx XML tree with
+    // 'PublicationDelivery' and 'dataObjects'
+    fn wrap_frame(&self, frame: Element, version_type: VersionType) -> Result<Element> {
+        let publication_timestamp = Element::builder("PublicationTimestamp")
+            .ns("http://www.netex.org.uk/netex/")
+            .append(Utc::now().to_rfc3339())
+            .build();
+        let participant_ref = Element::builder("ParticipantRef")
+            .ns("http://www.netex.org.uk/netex/")
+            .append(self.participant_ref.as_str())
+            .build();
+        let data_objects = Element::builder("dataObjects")
+            .ns("http://www.netex.org.uk/netex/")
+            .append(frame)
+            .build();
+        let root = Element::builder("PublicationDelivery")
+            .ns("http://www.netex.org.uk/netex/")
+            .attr("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance")
+            .attr("version", format!("1.09:FR-NETEX_{}-2.1-1.0", version_type))
+            .append(publication_timestamp)
+            .append(participant_ref)
+            .append(data_objects)
```

Comment:
I'm not sure but this node is not wrapped by `<CompositeFrame><frames></frames></CompositeFrame>`?

Reply:
This is still something to discuss with @prhod. From what I understood of NeTEx specification (well, the XSD files), you need to wrap in a `CompositeFrame` only if there is more than one frame. In the fixtures we do have for IDFM, they have 2 `GeneralFrame` but only one contains the stops. The other one seems useless and we might not produce it for NeTEx France. So if we do have only one to produce, we might just skip this extra level of `CompositeFrame`. But as I said, still to be discussed with @prhod.

Reply:
So, decision has been taken to go for the simplest solution for now (without a `CompositeFrame`). We're not 100% sure it's completely correct yet but it should fairly easy to wrap it in a `CompositeFrame` later if needed.